indigo_app/static/stylesheets/_akn.scss

```diff
@@ -2,6 +2,7 @@
 @import 'lib/indigo-web/scss/indigo-web';
 
 /* overrides for indigo inserts into akoma-ntoso trees */
+la-akoma-ntoso .ig,
```

Comment:
I consider this a custom style indigo-web

Reply:
It's safe for the `akoma-ntoso .ig` just below to fall away. We use the `.ig` to indicate "foreign" elements we've injected into the akoma-ntoso HTML. We only do that in the document editor view, which now uses la-akoma-ntoso entirely.